src/index.ts

```diff
@@ -806,11 +878,43 @@ function FlatpickrInstance(
       }
     }
 
-    self.changeMonth(loopDelta);
+    checkAvailableDays(() => {
+      self.changeMonth(loopDelta);
+    }, delta);
```

Comment:
Deltaen som sendes inn her blir feil i blant tror jeg. Når jeg bruker piltastene for å velge dato (uten skjermleser) og går opp eller ned til en måned som ikke har dato blir måned og år feil i beskjeden: 
![image](https://user-images.githubusercontent.com/468796/131469670-56aa2fc3-e88f-463f-9dbc-5a345091faa6.png)
Gjorde en kjapp debug og deltaen så ut til å være 7 og ikke 1 som forventet i checkAvailableDays?

Reply:
Fikset :-)